scripts/build

```diff
@@ -1,13 +1,4 @@
 #!/usr/bin/env bash
 
-yarn prep
-yarn build
-
-mkdir -p build/short-ext
-
-mv build/background.js build/short-ext/
-cp build/manifest.json build/short-ext/
-cp -r icons build/short-ext/
-
-cd build || exit
-zip -r short-ext.zip short-ext/* -x "*.DS_Store"
```

Comment:
Sometimes we may want to debug the `.zip` file locally. If we keep the same script for prod and dev, if something goes wrong, we can check the `zip` locally.

Reply:
Ohkay, I'll do something like `./scripts/build --prod` for conditionally calling `yarn:production` and creating the zip?
We still need `yarn:development` and `yarn:production` so as to not populate the env variables which are populated in build time, like the case of `GIT_TAG`.

Reply:
@rohithbalaji123 How about we eliminate this file and completely switch to package.json to manage the scripts?

Reply:
Yeah, that would be good too. :+1: